build/_k-standard-goals.shade

```diff
@@ -83,26 +83,34 @@ default Configuration='${E("Configuration")}'
 
 #native-compile target='compile' if='!IsMono && Directory.Exists(Path.Combine(BASE_DIR, "src"))'
   var programFilesX86 = '${Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}'
-  var msbuild = '${Path.Combine(programFilesX86, "MSBuild", "12.0", "Bin", "MSBuild.exe")}'
   var nativeProjects ='${Files.Include(Path.Combine(BASE_DIR, "src", "**", "*.vcxproj"))}'
 
   @{
-      if (nativeProjects.Any())
+    if (nativeProjects.Any())
```

Comment:
Then I am not going to separate it. Let me know if it is good to go and I will merge it in.